archunit/src/main/java/com/tngtech/archunit/core/importer/DomainBuilders.java

```diff
@@ -931,14 +909,17 @@ static <PARAMETER, VALUE> Set<VALUE> build(
                     Set<? extends BuilderWithBuildParameter<PARAMETER, ? extends VALUE>> builders,
                     PARAMETER parameter,
                     ImportedClasses importedClasses) {
+                return build(builders.stream(), parameter, importedClasses);
```

Comment:
I don't think I fully get it :thinking: Do you mean add `checkNotNull(builders)` here? But that would just throw an NPE, same as `builders.stream()`, no? And this line is not really ambiguous in dereferencing, so that it would hide the culprit?

Reply:
Oh, right! Never mind; I did not realize that `checkNotNull` throws a NPE, but just noticed that old code had this check... 🤦‍♂️